app/services/katello/pulp3/repository.rb

```diff
@@ -194,7 +194,7 @@ def delete(href = repository_reference.try(:repository_href))
       end
 
       def sync
-        sync_url_params = {remote: repo.remote_href, mirror: repo.root.mirror_on_sync}
+        sync_url_params = {remote: repo.remote_href, mirror: repo.root.mirror_on_sync, optimize: false}
```

Comment:
actually, will this work for non-yum repos?  (will it be ignored)?

Reply:
Doesn't look like:  `optimize` is not a valid attribute in `PulpFileClient::RepositorySyncURL`. Please check the name to make sure it's valid. List of attributes: [:remote, :mirror]


I think we need to only add it if its a yum type

Reply:
Waiting for tests to fail to find out and re-record.. :)

Reply:
Yes..needs to be for only rpm..Updating..